scripts/azureml-assets/azureml/assets/validate_assets.py

```diff
@@ -227,11 +227,27 @@ def validate_name(asset_config: assets.AssetConfig) -> int:
     return error_count
 
 
+def validate_categories(asset_config: assets.AssetConfig) -> int:
+    """Validate asset categories.
+
+    Args:
+        asset_config (AssetConfig): Asset config.
+
+    Returns:
+        int: Number of errors.
+    """
+    if len(asset_config.categories) == 0:
+        _log_error(asset_config, "Categories not found")

```

Comment:
We should also point to some documentation so that the engineer will know how to add categories. I'm updating the documentation right now and will get back to you shortly.

Reply:
I think this would be a good link: https://github.com/Azure/azureml-assets/wiki/Assets#assetyaml

Reply:
I was thinking you could do something like this: https://github.com/Azure/azureml-assets/blob/main/scripts/azureml-assets/azureml/assets/validate_assets.py#L337-L339

Reply:
ah that makes sense, will make that edit!